tools/trace2syz/main.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/google/syzkaller/pkg/db"
+	"github.com/google/syzkaller/pkg/hash"
+	"github.com/google/syzkaller/pkg/log"
+	"github.com/google/syzkaller/prog"
+	"github.com/google/syzkaller/sys"
+	"github.com/shankarapailoor/moonshine/trace2syz/trace2syz"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+var (
+	flagFile      = flag.String("file", "", "file to parse")
+	flagDir       = flag.String("dir", "", "director to parse")
+	flagTraceType = flag.String("traceType", trace2syz.Strace, "Type of trace e.g., strace, perf")
+)
+
+const (
+	OS               = "linux" //Target OS
+	Arch             = "amd64" //Target architecture
+	currentDBVersion = 3       //Marked as minimized
+)
+
+func main() {
+	rev := sys.GitRevision
+	flag.Parse()
+	target, err := prog.GetTarget(OS, Arch)
+	if err != nil {
+		log.Fatalf("error getting target: %v, git revision: %v", err.Error(), rev)
```

Comment:
Hmm.. so this is a "hack" because without importing the ```sys``` package, my target is empty (no syscall map, etc).  ```sys`` needs to call ```RegisterTarget``` with variables that are local to the OS, arch. Is there a cleaner way or should I keep as is, but add a comment?

Reply:
You can import it as:
```
    _ "github.com/google/syzkaller/sys"
```

Reply:
Oh cool. thanks!